app/models/spree/order_decorator.rb

```diff
@@ -191,5 +191,48 @@ def flow_payment_method
         'cc' # creait card is default
       end
     end
+
+    def flow_customer_email
+      flow_data['order'].dig('customer', 'email')
+    end
+
+    def flow_ship_address
+      flow_destination = flow_data['order'].dig('destination')
```

Comment:
No reason for a `dig` here:
```suggestion
      flow_destination = flow_data['order']['destination']
```

Reply:
Thanks! I missed the placement :facepalm: haha